### DISCUSSION TITLE
Options Picker Selected Label Extraction

### DISCUSSION THREAD

**Discussion by StanMalone** [Budibase/budibase#13202] [Help] [answered]
How can i extract the value from an options picker selected label, rather than it's selected value?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @StanMalone,

Here is a workaround I created a while back. I don't believe an enhancement was ever created. Hopefully this helps you. https://github.com/Budibase/budibase/discussions/10950#discussioncomment-6218976
